zzzz_modular_occulus/code/game/objects/items/weapons/implant/implants/carrion/carrion_spiders.dm

```diff
@@ -0,0 +1,50 @@
+/obj/item/weapon/implant/carrion_spider/explosive/install()
+	..()
+	spawn(600) //This is a 1 minute warning. I am varying these quite a lot
+	to_chat(wearer, SPAN_WARNING("You feel a sharp stabbing pain that slowly fades."))
+
+/obj/item/weapon/implant/carrion_spider/flashbang/install()
+	..()
+	spawn(3000) //5 minute warning
+	to_chat(wearer, SPAN_WARNING("You feel a sharp stabbing pain that slowly fades."))
+
+/obj/item/weapon/implant/carrion_spider/infection/install()
+	..()
+	spawn(50) //5 second warning, This one is kinda important to have feedback   
+	to_chat(wearer, SPAN_WARNING("You feel a sharp stabbing pain that slowly fades."))
+
+/obj/item/weapon/implant/carrion_spider/breeding/install()
+	..()
+	spawn(6000) //10 minute warning, It can only activate when a person is dead, so its not high priority
+	to_chat(wearer, SPAN_WARNING("You feel a sharp stabbing pain that slowly fades."))
+
+/obj/item/weapon/implant/carrion_spider/toxicbomb/install()
+	..()
+	spawn(600) //1 minute warning, potentially more deadly than an explosive spider, but unreliable
+	to_chat(wearer, SPAN_WARNING("You feel a sharp stabbing pain that slowly fades."))
+
+/obj/item/weapon/implant/carrion_spider/disruptor/install()
+	..()
+	spawn(3000) //5 minute warning.
+	to_chat(wearer, SPAN_WARNING("You feel a sharp stabbing pain followed by an odd electric tingle."))
+
+/obj/item/weapon/implant/carrion_spider/mindboil/Process()
+	..()
```

Comment:
This will double up the mindboil effects.

Reply:
oh thanks for catching that.